src/detection/card/style.js

```diff
@@ -0,0 +1,13 @@
+import { useMediaQuery, useTheme } from '@material-ui/core'
+
+import { useDesktopStyle } from 'detection/card/desktop-style'
+import { useMobileStyle } from 'detection/card/mobile-style'
+
+export const useStyles = () => {
+  const theme = useTheme()
+  const { root: desktopScreen } = useDesktopStyle()
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125920206-23357a6f-bad9-475b-b1b0-c0a2ff65cfa7.png)
Let's not destruct the `root` property here, so that we can use the `classes` prop instead of the `className` prop in our `<Wrapper/>` component.